containers/daap-create-metadata/src/var/task/create_metadata.py

```diff
@@ -35,7 +35,7 @@ def handler(event, context):
 
     data_product_metadata = DataProductMetadata(data_product_name, logger)
 
-    if not data_product_metadata.metadata_exists:
+    if data_product_metadata.metadata_exists is False:
```

Comment:
```suggestion
    if not data_product_metadata.metadata_exists:
```
Original is more standard python (`if not .exists`), and more typically what I'd expect.

If insisting on moving to comparing with False, use ` == False`

Reply:
😢 ok